src/components/sideBarContentEdit/SidebarContentEdit.tsx

```diff
@@ -179,7 +179,7 @@ const SidebarContentEdit = ({
           </Text>
           <OnlySelectField
             name="speakers"
-            editedData={sideBarData.list.speakers}
+            editedData={sideBarData.list.speakers || []}
```

Comment:
you can use optional chaining here. if `list` is undefined, it would throw an error.
`list` is presumably never undefined but just to be safe
```suggestion
            editedData={sideBarData.list?.speakers || []}
```
and same for the rest

Reply:
if it's presumable never undefined, then isn't better to not add that? so if it's ever undefined then it will be apparent to us that something is not working as it is supposed to be working and is not.

Reply:
> if it's presumable never undefined, then isn't better to not add that? so if it's ever undefined then it will be apparent to us that something is not working as it is supposed to be working and is not.

you do have a point. I looked for where we get the data from and we do explicitly define it [here](https://github.com/bitcointranscripts/transcription-review-front-end/blob/e4167d8d380b32656244509653f7c9a488afb8d3/src/components/transcript/index.tsx#L78).

Given that it also follows the type `SidebarData` then it is fine. If anything is wrong, it would be from the Transcript content itself from the backend